src/main/resources/db/changelog/v.0.1/db.changelog-internship.yaml

```diff
@@ -0,0 +1,51 @@
+databaseChangeLog:
+  - changeSet:
+      id: 31032021-add-table-internship
+      author: olga
+      changes:
+        - createTable:
+            tableName: internship
+            columns:
+              - column:
+                  name: inship_id
+                  type: UUID
+                  constraints:
+                    primaryKey: true
+              - column:
+                  name: inship_name
+                  type: VARCHAR(50)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: inship_start_date
+                  type: TIMESTAMP
+              - column:
+                  name: inship_end_date
+                  type: TIMESTAMP
+              - column:
+                  name: inship_publication_date
+                  type: TIMESTAMP
+              - column:
+                  name: inship_tech_skills
+                  type: VARCHAR(255)
+                  constraints:
+                    nullable: false
```

Comment:
Remove please non nullable constraint for fields `inship_tech_skills` `inship_description` and `inship_requirements`.
As far as I remember Zhanna said that internship can be created without additional information.
If in future we recognize that some field should be required appropriate constraint can be added.

Reply:
done